torch_np/tests/numpy_tests/core/test_numeric.py

```diff
@@ -3002,12 +2793,12 @@ def test_nd(self, nd):
 
         # only one
         x[...] = False
-        x.flat[0] = True
+        x.ravel()[0] = True
```

Comment:
What's the reason for this change?

Reply:
Shrug.
A technical reason is that ravel is implemented and flat is not. 
For this test specifically, there is no difference, the test passes in both pristine numpy and our `torch_np`.

Going forward, I have a feeling that `ravel()` is a common thing to do, while `.flat` is mainly from very old code, so I'd rather push it into "yeah, implement some day" bucket.

Reply:
cool.